mods/ra/maps/sarin-gas-1-crackdown/crackdown.lua

```diff
@@ -42,27 +42,21 @@ SetupTriggers = function()
 		greece.MarkCompletedObjective(objDestroyAllTrucks)
 	end)
 
-	Trigger.OnEnteredFootprint({ TruckEscapeCenter.Location }, function(actor, triggerlose1)
-		if actor.Owner == ussr and actor.Type == "truk" then
-			Trigger.RemoveProximityTrigger(triggerlose1)
-			actor.Destroy()
-			greece.MarkFailedObjective(objDestroyAllTrucks)
-		end
-	end)
-
-	Trigger.OnEnteredFootprint({ EscapeNorth10.Location }, function(actor, triggerlose2)
-		if actor.Owner == ussr and actor.Type == "truk" then
-			Trigger.RemoveProximityTrigger(triggerlose2)
-			actor.Destroy()
-			greece.MarkFailedObjective(objDestroyAllTrucks)
-		end
-	end)
+	SetTrukTrigger(TruckEscapeCenter.Location, triggerlose1)
+	SetTrukTrigger(EscapeNorth10.Location, triggerlose2)
+	SetTrukTrigger(EscapeSouth5.Location, triggerlose3)
```

Comment:
Generally, I don't think we want to set three triggers here, but just set one trigger that has all three locations:
`Trigger.OnEnteredFootprint({ a, b, c }, function...`
(We don't need an extra method then as well.)

Reply:
Thank you! I'll make the changes early next week.